### PR TITLE
Fix tapChanger modification bugs

### PR DESCRIPTION
**Pull request by ayolab**

### CODE REVIEW

**Review by Meklo — APPROVED** [review]

File: src/main/java/org/gridsuite/modification/server/modifications/TwoWindingsTransformerModification.java [hunk 1]
```diff
@@ -438,29 +459,88 @@ private void addPhaseTapChangersToTwoWindingsTransformer(Network network, TwoWin
...
+                phaseTapChangerSubreporter.report(Report.builder()
+                            .withKey(TapChangerType.PHASE.name())
+                            .withDefaultMessage(PHASE_TAP_CHANGER_SUBREPORTER_DEFAULT_MESSAGE)
+                            .withSeverity(TypedValue.INFO_SEVERITY)
+                            .build());
+            }
             ModificationUtils.getInstance().reportModifications(phaseTapChangerSubreporter, regulationReports, regulationMode != null ? regulationMode.name() : null, ModificationUtils.getInstance().formatRegulationModeReport(regulationMode));
+            ModificationUtils.getInstance().reportModifications(phaseTapChangerSubreporter, positionsAndStepsReports, "phaseTapChangerPositionsAndStepsModification", "    Tap Changer");
         }
+    }
```
Comment (dbraquart) [inline src/main/java/org/gridsuite/modification/server/modifications/TwoWindingsTransformerModification.java:503]: could you specify the scope of the code to be removed ?
do we we have to remove everything from TODO to the end of this function ?

File: src/main/java/org/gridsuite/modification/server/modifications/TwoWindingsTransformerModification.java [hunk 2]
```diff
@@ -346,15 +333,48 @@ private void addRatioTapChangersToTwoWindingsTransformer(Network network, TwoWin
...
+                ratioTapChangerReporter = subReporter.createSubReporter(TapChangerType.RATIO.name(), RATIO_TAP_CHANGER_SUBREPORTER_DEFAULT_MESSAGE);
+                ratioTapChangerReporter.report(Report.builder()
+                            .withKey(TapChangerType.RATIO.name())
+                            .withDefaultMessage(RATIO_TAP_CHANGER_SUBREPORTER_DEFAULT_MESSAGE)
+                            .withSeverity(TypedValue.INFO_SEVERITY)
+                            .build());
+            }
+            ModificationUtils.getInstance().reportModifications(ratioTapChangerReporter, voltageRegulationReports, "ratioTapChangerVoltageRegulationModification", "    Voltage regulation");
+            ModificationUtils.getInstance().reportModifications(ratioTapChangerReporter, positionsAndStepsReports, "ratioTapChangerPositionsAndStepsModification", "    Tap Changer");
+        }
```
Comment (dbraquart) [inline src/main/java/org/gridsuite/modification/server/modifications/TwoWindingsTransformerModification.java:377]: could you specify the scope of the code to be removed ?
do we we have to remove everything from TODO to the end of this function ?

**Review by dbraquart — CHANGES_REQUESTED** [review]
code: ok - just better identify the to-be-removed-code
tests: ok

**Review by dbraquart — APPROVED** [review]
code: ok
tests:ok

**Pull request merged**